docs/source/tutorial/understanding_evaluation.rst

```diff
@@ -180,6 +180,51 @@ may be additional known triples :math:`(h, r, t')` for :math:`t \neq t'`. If the
 filtered evaluation setting ignores for a given triple :math:`(h, r, t)` the scores of all other *known* true triples
 :math:`(h, r, t')`.
 
+PyKEEN provides training and evaluation workflows for which the set of positive triples is pre-defined. When using
+the :func:`pykeen.hpo.hpo_pipeline` or :func:`pykeen.pipeline.pipeline`, PyKEEN differentiates two scenarios in which
+the set of *known* true triples is built differently according to [bordes2013]_.
+During hyper-parameter optimization (HPO) and early stopping, the set of *known* true triples consists per default of
+the training and validation set. Only during the evaluation of the final model on the test set, it consists of training,
+validation, and test triples. We explicitly do not use test triples for filtering during HPO and early stopping in
+order to avoid any test leakage. In case the validation triples should *not* be filtered when evaluating the test
+dataset, the argument ``filter_validation_when_testing=False`` can be passed to the two functions above.
+
+Please note that when using an :class:`pykeen.evaluation.Evaluator` directly in your custom workflow to evaluate your
```

Comment:
does it make sense to introduce writing custom workflows in the tutorial section of the docs, whose purpose is more to inform the reader what's going on? Why not give a code example that uses the pipeline used the default and the non-default way, then keep this in the reference section?

I know I said in a previous comment that it would be good to de-duplicate the tutorial's "Understanding the Evaluation" and the reference for evaluation, but I think the way that materialized still needs some work

Reply:
I hope it doesn't seem like I'm just being arbitrary - every time we do an update, I've gone and built the docs locally with

```shell
cd docs/
make html
open build/html/index.html
```

and read through the pages from top to bottom to think how a new reader might find this information and what they might think of it, assuming they're going from the beginning of the tutorial to the end in order (which is what everyone does)

Reply:
Addressed in 7ba9969e